config/ext_jenkins-infra-jobs.yaml

```diff
@@ -123,6 +123,61 @@ jobsDefinition:
       jenkins-wiki-exporter:
         name: Wiki Exporter
         jenkinsfilePath: Jenkinsfile
+  reports:
+    name: Reports
+    description: Folder hosting all the reporting tasks jobs
+    kind: folder
+    children:
+      artifactory-users-report:
+        name: "Artifactory Users Report"
+        repository: "infra-reports"
+        jenkinsfilePath: "artifactory-users-report/Jenkinsfile"
+        credentials:
+          artifactoryAdmin:
+            description: "Artifactory API for updating permissions"
+            username: "${ARTIFACTORY_API_INFRA_REPORTS_USERNAME}"
+            usernameSecret: true
+            password: "${ARTIFACTORY_API_INFRA_REPORTS_PASSWORD}"
+      fork-report:
+        name: "GitHub Forks Report"
```

Comment:
```suggestion
        name: "Github Forks Report"
```

Reply:
GitHub is the brand name,  don't change this